### PR TITLE
feat(adm): adding feature for the admin page

### PR DESCRIPTION
**Pull request by Raafm**
## :memo: Summary
adding feature with 4  scenario for the admin page

## :white_check_mark: Type of change

- [x] Feature
- [ ] Refactor
- [ ] Bug fix

### CODE REVIEW

File: features/calculateTotalPurchase.feature [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Feature: Estatísticas
+
```
Comment (aalmds) [inline features/calculateTotalPurchase.feature:6]: Missing As a ... I want to ... So that ...

File: features/calculateTotalPurchase.feature [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Feature: Estatísticas
+
+    Scenario: Ver estatÍstica geral
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
```
Comment (aalmds) [inline features/calculateTotalPurchase.feature]:
```suggestion
        Given que o usuário com CPF “00000000013” está na página de "administrador"
```

File: features/calculateTotalPurchase.feature [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    Scenario: Ver estatÍstica geral
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras (no tempo) são respectivamente "15000", "20000" e "10000"
+        Then os valores  "15000", "20000" e "10000" que são, respectivamente, a média, o máximo e o mínimo do valor das compras (no tempo) são mostrados
+
+    Scenario: Ver estatística por pedido
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
```
Comment (aalmds) [inline features/calculateTotalPurchase.feature]:
```suggestion
        Given que o usuário com CPF “00000000013” está na página de "administrador"

```

File: features/calculateTotalPurchase.feature [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        And A média, o máximo e o mínimo do valores das compras (por pedido) são respectivamente "150", "200" e "100"
+        When  O usuário acessa a aba de dados por pedido (ID 130)
```
Comment (aalmds) [inline features/calculateTotalPurchase.feature]:
```suggestion
        When  o usuário de CPF "00000000013" acessa os "dados por pedido"
```

File: features/calculateTotalPurchase.feature [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras canceladas (por pedido) são respectivamente "150", "200" e "100"
+        When  O usuário acessa a aba de dados para pedidos cancelados (ID 131)
+        Then os valores "150", "200" e "100" que são, respectivamente, a média, o máximo e o mínimo dos valores das compras canceladas por pedido são mostrados
+
+    Scenario: Selecionar período
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        When  O usuário seleciono o período  "MÊS"
+        And O usuário acessa a aba de dados por pedido (ID 132)
```
Comment (aalmds) [inline features/calculateTotalPurchase.feature]:
```suggestion
        Given que o usuário com CPF “00000000013” está na página de "administrador"
        When o usuário de CPF "00000000013" seleciona o período "mês"
        And o usuário de CPF "00000000013"  acessa os "dados por pedido"
```

File: features/calculateTotalPurchase.feature [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras (por pedido) são respectivamente "150", "200" e "100"
+        When  O usuário acessa a aba de dados por pedido (ID 130)
+        Then os valores "150", "200" e "100" que são, respectivamente, a média, o máximo e o mínimo dos valores das compras em cada pedidos são mostrados
+
+    Scenario: Ver estatística dos cancelados
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras canceladas (por pedido) são respectivamente "150", "200" e "100"
+        When  O usuário acessa a aba de dados para pedidos cancelados (ID 131)
```
Comment (aalmds) [inline features/calculateTotalPurchase.feature]:
```suggestion
        When o usuário  de CPF "00000000013" acessa as dados para "pedidos cancelados"
```

**Review by aalmds — CHANGES_REQUESTED** [review]

File: features/calculateTotalPurchase.feature [hunk 7]
```diff
@@ -0,0 +1,28 @@
+Feature: Estatísticas
+
+    Scenario: Ver estatÍstica geral
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras (no tempo) são respectivamente "15000", "20000" e "10000"
```
Comment (acrucha) [inline features/calculateTotalPurchase.feature]: Just to make it easier to understand, you can replace it with:

```
And as métricas de valores "dos pedidos no tempo" são "15000" para "média", "20000" para "valor máximo" e "10000" para "valor mínimo"
```

File: features/calculateTotalPurchase.feature [hunk 8]
```diff
@@ -0,0 +1,28 @@
+Feature: Estatísticas
+
+    Scenario: Ver estatÍstica geral
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras (no tempo) são respectivamente "15000", "20000" e "10000"
+        Then os valores  "15000", "20000" e "10000" que são, respectivamente, a média, o máximo e o mínimo do valor das compras (no tempo) são mostrados
```
Comment (acrucha) [inline features/calculateTotalPurchase.feature]: Also:

```
And as métricas de valores "dos pedidos no tempo" mostradas são "15000" para "média", "20000" para "valor máximo" e "10000" para "valor mínimo"
```

File: features/calculateTotalPurchase.feature [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+    Scenario: Ver estatÍstica geral
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras (no tempo) são respectivamente "15000", "20000" e "10000"
+        Then os valores  "15000", "20000" e "10000" que são, respectivamente, a média, o máximo e o mínimo do valor das compras (no tempo) são mostrados
+
+    Scenario: Ver estatística por pedido
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras (por pedido) são respectivamente "150", "200" e "100"
```
Comment (acrucha) [inline features/calculateTotalPurchase.feature]: As I suggested before:
```
And as métricas de valores "por pedido" são "150" para "média", "200" para "valor máximo" e "100" para "valor mínimo"
```

File: features/calculateTotalPurchase.feature [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+        When  O usuário acessa a aba de dados por pedido (ID 130)
+        Then os valores "150", "200" e "100" que são, respectivamente, a média, o máximo e o mínimo dos valores das compras em cada pedidos são mostrados
```
Comment (acrucha) [inline features/calculateTotalPurchase.feature]:
```
Then as métricas de valores "por pedido" mostradas são "150" para "média", "200" para "valor máximo" e "100" para "valor mínimo"
```

File: features/calculateTotalPurchase.feature [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+    Scenario: Ver estatística dos cancelados
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras canceladas (por pedido) são respectivamente "150", "200" e "100"
```
Comment (acrucha) [inline features/calculateTotalPurchase.feature]: You need to adjust this step to be in the same format as the others that I commented before. You just need to copy the new step format and change the statistics type (in this case, "de pedidos cancelados").

File: features/calculateTotalPurchase.feature [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+        When  O usuário acessa a aba de dados para pedidos cancelados (ID 131)
+        Then os valores "150", "200" e "100" que são, respectivamente, a média, o máximo e o mínimo dos valores das compras canceladas por pedido são mostrados
```
Comment (acrucha) [inline features/calculateTotalPurchase.feature]:
```
Then as métricas de valores "de pedidos cancelados" mostradas são "150" para "média", "200" para "valor máximo" e "100" para "valor mínimo"
```

File: features/calculateTotalPurchase.feature [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+
+    Scenario: Selecionar período
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        When  O usuário seleciono o período  "MÊS"
+        And O usuário acessa a aba de dados por pedido (ID 132)
+        Then os valores "150", "200" e "100" que são, respectivamente, a média, o máximo e o mínimo dos valores das compras em cada pedidos no mês atual são mostrados
```
Comment (acrucha) [inline features/calculateTotalPurchase.feature]:
```
Then as métricas de valores "por pedido" mostradas são "150" para "média", "200" para "valor máximo" e "100" para "valor mínimo"
And o período selecionado é "mês"
```

File: features/calculateTotalPurchase.feature (hunk 7 above)
Comment (acrucha) [reply features/calculateTotalPurchase.feature]: I chose the word "métricas" to refer to the statistics values, but you can choose another word. If you choose a different word, please remember to use it in all scenarios. All steps of this type must be identical, with the only difference being the parameters (the strings you passed such as "15000", "média", "valor máximo"...)

Comment (acrucha) [reply features/calculateTotalPurchase.feature]: When it comes to verifying that these values are correct, you can make a request to the backend. You can use [this](https://docs.cypress.io/api/commands/request) to receive the data in Cypress (test framework).

Comment (acrucha) [reply features/calculateTotalPurchase.feature]: If you have any doubts, you can always send me a message, and I will get back to you as soon as possible.

**Review by acrucha — CHANGES_REQUESTED** [review]

**Review by Raafm — COMMENTED** [review]
fix(adm): name stats metrics

**Pull request closed without merging**